docs/ref/models/querysets.txt

```diff
@@ -2562,7 +2562,7 @@ Case-insensitive starts-with.
 
 Example::
 
-    Entry.objects.filter(headline__istartswith='will')
+    Entry.objects.filter(headline__istartswith='Will')
```

Comment:
`iendswith` is using lower case in both places so maybe it's better to change this SQL equivalent instead?